sycl/CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ set(SYCL_MINOR_VERSION 7)
 set(SYCL_PATCH_VERSION 0)
 # Don't forget to re-enable sycl_symbols_windows.dump once we leave ABI-breaking
 # window!
-set(SYCL_DEV_ABI_VERSION 8)
+set(SYCL_DEV_ABI_VERSION 9)
```

Comment:
I think I saw this change already in another ABI breaking change today, so someone will get a conflict.

Reply:
Yeah, that goes for all ABI break PRs with the way we handle incrementing `SYCL_DEV_ABI_VERSION` right now.